kivy/core/window/_window_sdl2.pyx

```diff
@@ -109,22 +109,10 @@ cdef class _WindowSDL2Storage:
         if USE_IOS:
             orientations = 'LandscapeLeft LandscapeRight Portrait PortraitUpsideDown'
 
-        # Override the Android orientation based on what p4a was built with, if available.
-        # This may be overridden again by $KIVY_ORIENTATION.
         if USE_ANDROID:
-            env_orientations = environ.get('P4A_ORIENTATION', orientations)
-
-            if env_orientations == 'portrait':
-                orientations = 'Portrait'
-            elif env_orientations == 'landscape':
-                orientations = 'LandscapeLeft'
-            elif env_orientations == 'sensor':
-                orientations = 'sensor'
-            else:
-                Logger.warning(('Could not satisfy Android orientation "{}", only '
-                               '{{portrait,landscape and sensor}} are currently supported. '
-                                'Defaulting to portrait').format(orientations))
-                orientations = 'Portrait'
+            # Do not hint anything: by default the value in the AndroidManifest.xml will be used
+            # Note that the user can still override this via $KIVY_ORIENTATION if they wish
+            orientations = ''
```

Comment:
Haven't tested it but @inclement has, and I use a similar approach in my non-kivy app and this looks like the right thing to do(TM) to me. :+1:

Reply:
I've tested it and confirm that it works.

¡¡¡Good work, @inclement and @JonasT!!! :+1: